src/main/java/frc/robot/commands/common/arm/ArmCommandWithButton.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.common.arm;
```

Comment:
This looks very similar to what https://github.com/DevilBotz2876/Reefscape2025/blob/main/src/main/java/frc/robot/commands/common/arm/ArmToPosition.java is doing? Take a look, compare,  and see if you agree.